patroni/ctl.py

```diff
@@ -680,6 +671,21 @@ def output_members(cluster, name, extended=False, fmt='pretty'):
 
     print_output(columns, rows, alignment, fmt)
 
+    service_info = []
+    if cluster.is_paused():
+        service_info.append('Maintenance mode: on')
+
+    if cluster.failover and cluster.failover.scheduled_at:
```

Comment:
Hm, does this mean there can be only one failover scheduled at any given moment?

Reply:
Yes